packages/instrumenter/src/mutation-level/default-mutation-levels.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
I think it's a good idea to separate this config, except I would expect it to be in a `.ts` file, so you can simply import it, instead of having to load it some other way.

Especially since the names used here should be in sync with the names in the stryker-core.json file. Having it as a JSON file here breaks type safety.

Reply:
Good observation! We moved the `default-mutation-levels` to a .ts file, and used the spread operator to remove some redundancy between the levels.